README.md

```diff
@@ -48,21 +181,33 @@ The following is a non-normative example of a W3C VC using the external [JWT pro
     ],
     "type": [
       "VerifiableCredential",
-      "UniversityDegreeCredential"
+      "IDCardredential"
     ],
     "credentialSubject": {
-      "degree": {
-        "type": "BachelorDegree",
-        "name": "Bachelor of Computer Science"
-      }
+      "given_name": "Fredrik",
+      "family_name": "Strömberg",
+      "birthdate": "1949-01-22",
+      "place_of_birth": {
+        "country": "SE",
+        "locality": "Örnsköldsvik"
+      },
+      "nationality": "SE",
+      "number": "4901224131",
+      "date_of_issuance":"2010-03-23",
+      "date_of_expiry":"2020-03-22"
+      "issuer":{
+        "name":"Skatteverket",
+        "country":"SE"
     }
   }
 }
 ```
 
 ## W3C Verifiable Credential using internal proofs
 
-The following is a non-normative example of a W3C VC using the internal proof format. The proof property contains a JSON-LD Proof and uses the detached JWS encoding for the signature representation:
+The following is a non-normative example of a W3C VC using the internal proof format. The proof property contains a JSON-LD Proof and uses the detached JWS encoding for the signature representation.
```

Comment:
Not sure is we want to align the VC spec's use of `external` and `embedded` as opposed to `internal` ?

https://www.w3.org/TR/2019/REC-vc-data-model-20191119/#proofs-signatures

Reply:
@awoie WDYT?

Reply:
let's fix that later. i agree that embedded and external are the correct terms.